modules/effects/src/effects_metadata.ts

```diff
@@ -2,7 +2,9 @@ import { EffectMetadata, EffectsMetadata } from './models';
 import { getCreateEffectMetadata } from './effect_creator';
 import { getEffectDecoratorMetadata } from './effect_decorator';
 
-export function getEffectsMetadata<T>(instance: T): EffectsMetadata<T> {
+export function getEffectsMetadata<T extends Object>(
```

Comment:
this may introduce a breaking change if the `getEffectsMetadata` function (that is part of the public API) is used with an argument of the wrong type